examples/dt/nfv/nfv-ovs-dpdk-sriov-2nodesets/edpm/nodeset/values.yaml

```diff
@@ -0,0 +1,228 @@
+---
+# yamllint disable rule:line-length
+
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-nodeset-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  root_password: cmVkaGF0Cg==
+  preProvisioned: false
+  baremetalSetTemplate:
+    ctlplaneInterface: eno2  # CHANGEME
+    cloudUserName: cloud-admin
+    provisioningInterface: enp1s0  # CHANGEME
+    bmhLabelSelector:
+      app: openstack  # CHANGEME
+    passwordSecret:
+      name: baremetalset-password-secret
+      namespace: openstack
+  ssh_keys:
+    # Authorized keys that will have access to the dataplane computes via SSH
+    authorized: CHANGEME
```

Comment:
missing # in CHANGEME instead # CHANGEME

Reply:
if i add it, i get this error
Error: accumulating components: accumulateDirectory: "recursed accumulation of path '/home/runner/work/architecture/architecture/va/nfv/ovs-dpdk-sriov/edpm/nodeset': accumulating components: accumulateDirectory: \"recursed accumulation of path '/home/runner/work/architecture/architecture/lib/dataplane/nodeset': fieldPath `data.ssh_keys.authorized` is missing for replacement source ConfigMap.[noVer].[noGrp]/edpm-nodeset-values.[noNs]\""
examples/dt/nfv/nfv-ovs-dpdk-sriov-2nodesets/edpm/nodeset/kustomization.yaml: ERROR
Error: accumulating components: accumulateDirectory: "recursed accumulation of path '/home/runner/work/architecture/architecture/va/nfv/ovs-dpdk-sriov/edpm/nodeset': accumulating components: accumulateDirectory: \"recursed accumulation of path '/home/runner/work/architecture/architecture/lib/dataplane/nodeset': fieldPath `data.ssh_keys.authorized` is missing for replacement source ConfigMap.[noVer].[noGrp]/edpm-nodeset-values.[noNs]\""

This  is a copy/paste from the ovs-dpdk-sriov va, and it is not there either
https://github.com/openstack-k8s-operators/architecture/blob/main/examples/va/nfv/ovs-dpdk-sriov/edpm/nodeset/values.yaml#L25

Reply:
Yes, because you need a `key: value` pair for the field to be present.  Doing something like `field: # CHANGEME` is not enough.  It would have to be `field: something # CHANGEME`.

Reply:
Yes, that it what I was pointing out.  The field must have a key and a value for it to be considered present, and a comment does not count as a value.